src/GoodsList.tsx

```diff
@@ -8,7 +8,7 @@ type Props = {
 export const GoodsList: React.FC<Props> = ({ goods }) => (
```

Comment:
https://github.com/mate-academy/react_dynamic-list-of-goods/blob/master/checklist.md